shell/common/persistent_cache.cc

```diff
@@ -80,23 +86,81 @@ sk_sp<SkData> PersistentCache::load(const SkData& key) {
   return SkData::MakeWithCopy(mapping->GetMapping(), mapping->GetSize());
 }
 
+static void PersistentCacheStore(fml::RefPtr<fml::TaskRunner> worker,
+                                 std::shared_ptr<fml::UniqueFD> cache_directory,
+                                 std::string key,
```

Comment:
I wonder why `std::string key` instead of `const std::string& key`? `std::move` may save us the copy, but we can type less `std::move` if we use `const std::string&`?

Reply:
Preventing unnecessary copies as a matter of habit. I don't mind typing the few extra characters (I guess thats subjective). Though I am pretty sure SSOs are already in effect for these particular key strings. But don't want to have to guess.